libcloudforensics/gcp.py

```diff
@@ -1048,6 +1147,65 @@ def Delete(self):
     self.project.BlockOperation(response)
 
 
+class GoogleComputeImage(GoogleComputeBaseResource):
+  """Class representing a Compute Engine Image.
+  Attributes:
+    disk (GoogleComputeDisk): Disk used for the Snapshot.
+  """
+
+  def GetOperation(self):
+    """Get API operation object for the image.
+
+    Returns:
+       dict: An API operation object for a Google Compute Engine Image.
+    """
+
+    gce_image_client = self.project.GceApi().images()
+    request = gce_image_client.get(
+        project=self.project.project_id, image=self.name)
+    response = request.execute()
+    return response
+
+  def ExportImage(self, gcs_output_folder, output_name=None):
+    """Export compute image to Google Cloud storage.
+
+    Exported image is compressed and stored in .tar.gz format.
+
+    Args:
+      gcs_output_folder (str): Folder path of the exported image.
+      output_name (str): Name of the output file, must end with .tar.gz,
+        if not exist, the [image_name].tar.gz will be used
```

Comment:
Add indent and new line before `Raises:`

Reply:
Done